src/System.Private.CoreLib/shared/System/DefaultBinder.cs

```diff
@@ -217,7 +218,7 @@ partial class DefaultBinder : Binder
                     // now do a "classic" type check
                     if (pCls.IsPrimitive)
                     {
-                        if (argTypes[paramOrder[i][j]] == null || !CanChangePrimitive(args[paramOrder[i][j]].GetType(), pCls))
+                        if (argTypes[paramOrder[i][j]] == null || !CanChangePrimitive(args[paramOrder[i][j]]?.GetType()!, pCls))
```

Comment:
This and below row 251 had a conflict, added question mark instead of bang to make same as row 251

Reply:
Actually `CanChangePrimitive` can take nulls. So I marked its parameters as nullable.